whatshap/cli/polyphase.py

```diff
@@ -449,35 +447,51 @@ def phase_single_individual(readset, phasable_variant_table, sample, phasing_par
             )
 
         genotype_slice = genotype_list[block_start:block_end]
-        clustering, path, haplotypes, cut_positions = phase_single_block(
+        clustering, path, haplotypes, cut_positions, haploid_cuts = phase_single_block(
             block_readset, genotype_slice, phasing_param, timers
         )
 
         blockwise_clustering.append(clustering)
         blockwise_paths.append(path)
         blockwise_haplotypes.append(haplotypes)
         blockwise_cut_positions.append(cut_positions)
+        blockwise_haploid_cuts.append(haploid_cuts)
 
     # Aggregate blockwise results
-    clustering, threading, haplotypes, cut_positions = aggregate_phasing_blocks(
+    clustering, threading, haplotypes, cut_positions, haploid_cuts = aggregate_phasing_blocks(
         block_starts,
         block_readsets,
         blockwise_clustering,
         blockwise_paths,
         blockwise_haplotypes,
         blockwise_cut_positions,
+        blockwise_haploid_cuts,
         phasing_param,
     )
 
-    # Write new VCF file
+    # Summarize data for VCF file
     accessible_positions = sorted(readset.get_positions())
     components = {}
+    haploid_components = {}
 
     ext_cuts = cut_positions + [num_vars]
     for i, cut_pos in enumerate(cut_positions):
         for pos in range(ext_cuts[i], ext_cuts[i + 1]):
             components[accessible_positions[pos]] = accessible_positions[ext_cuts[i]]
             components[accessible_positions[pos] + 1] = accessible_positions[ext_cuts[i]]
+            haploid_components[accessible_positions[pos]] = [0 for _ in range(phasing_param.ploidy)]
```

Comment:
Use `[0] * phasing_param.ploidy` to create this list (and in the next line as well)